corio.py

```diff
@@ -263,9 +263,10 @@ def main(options):
 
     :param options: Parsed Arguments.
     """
-    if options.degraded_mode:
-        activate_degraded_mode(options)
-        options.number_of_nodes -= literal_eval(os.getenv("DEGRADED_PODS_CNT"))
+    LOGGER.info("Setting up environment!!")
```

Comment:
pls remove line 266-269 as this is part of pre_requisite.

Reply:
Done. Please check now